akka-docs/src/main/paradox/typed/cluster-concepts.md

```diff
@@ -31,7 +31,7 @@ and membership state transitions.
 ### Gossip
 
 The cluster membership used in Akka is based on Amazon's [Dynamo](https://www.allthingsdistributed.com/files/amazon-dynamo-sosp2007.pdf) system and
-particularly the approach taken in Basho's' [Riak](https://riak.com/technology/architecture/) distributed database.
+particularly the approach taken in Basho's' [Riak](http://riak.com/technology/architecture/) distributed database.
```

Comment:
Que?

Reply:
Replaced it with the Wikipedia page link instead, as their cert setup is broken.